platform/kobo/koreader.sh

```diff
@@ -166,11 +201,27 @@ RETURN_VALUE=85
 while [ $RETURN_VALUE -eq 85 ]; do
     # Do an update check now, so we can actually update KOReader via the "Restart KOReader" menu entry ;).
     ko_update_check
+    # Do the fb depth switch, unless it's been disabled
+    ko_do_fbdepth
+    # Check if we wanted to enable debug logging...
+    DEV_ARGS=""
```

Comment:
Since we start with our settings before even parsing the command line arguments, this solution strikes me as somewhat overcomplicated?

https://github.com/koreader/koreader/blob/cfa946adbd8f5d52cb45b908e1735b5bd5570351/reader.lua#L29-L30

https://github.com/koreader/koreader/blob/cfa946adbd8f5d52cb45b908e1735b5bd5570351/reader.lua#L82-L85

Reply:
Yeah, that makes more sense, I wasn't aware of the reader.lua bit when I came up with that ;).